changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
+  - falconctl_options
+  - falcon_kernel_check
+  - cryptoinfo
+  - cryptsetup_status
+  - parse_plist
+  - parse_json
+  - parse_jsonl
+  - parse_xml
+  - parse_ini
+  - dev_table_tooling
+  - dsim_default_associations
+  - filevault_status
+  - firefox_preferences
+  - firmwarepasswd
+  - gsettings
+  - gsettings_metadata
```

Comment:
```suggestion
```

Noah: Let's cut these for now. I think `gsettings` and `gsettings_metadata` should be one `gsettings` table. We can add it later.